packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/programmableKeys/telecom-telephony-line-phone-programmableKeys.controller.js

```diff
@@ -97,7 +97,17 @@ export default class TelecomTelephonyLinePhoneProgammableKeysCtrl {
     return this.line.getPhone().then(() => {
       if (this.line.hasPhone) {
         return this.line.phone.initDeffered().then(() => {
-          this.functionKeys.raw = cloneDeep(this.line.phone.functionKeys);
+          const re = /^SUP[0-9]$/;
+          this.functionKeys.raw = this.line.phone.functionKeys.map((functionKey) => {
+            const key = clone(functionKey);
+            if (key.function.match(re)) {
```

Comment:
It will probably raise an error like: `TypeError: Cannot read property 'match' of null`.

`key.function` must be at least as type String.